privacyidea/api/token.py

```diff
@@ -673,6 +673,31 @@ def setpin_api(serial=None):
     return send_result(res)
 
 
+@token_blueprint.route('/description', methods=['POST'])
+@token_blueprint.route('/description/<serial>', methods=['POST'])
+@prepolicy(check_base_action, request, action=ACTION.SETDESCRIPTION)
+@event("token_set", request, g)
+@log_with(log)
+def set_description_api(serial=None):
```

Comment:
Till now there is only one policy ``ACTION.SET`` that allows an admin to change all settings of a token.
We do not want to get there into more details. Users till now were not able to change anything.
THis is why we add a new endpoint to change the description.